libsplinter/src/biome/user/store/diesel/mod.rs

```diff
@@ -42,15 +42,15 @@ impl SplinterUserStore {
 }
 
 impl UserStore<SplinterUser> for SplinterUserStore {
-    fn add_user(&self, user: SplinterUser) -> Result<(), UserStoreError> {
+    fn add_user(&mut self, user: SplinterUser) -> Result<(), UserStoreError> {
```

Comment:
I'm not sure that the commit message makes sense in the context of this PR (it makes more sense in the context of the original PR).